posts/2021-11-26-jakarta-ee-9.1.adoc

```diff
@@ -13,19 +13,16 @@ open-graph-image: https://openliberty.io/img/twitter_card.jpg
 = Jakarta EE 9.1 support and configurable response headers in Open Liberty 21.0.0.12
 Ryan Storey <https://github.com/ryan-storey>
 :imagesdir: /
-:url-prefixx
+:url-prefix:
 :url-about: /
 //Blank line here is necessary before starting the body of the post.
 
 Open Liberty 21.0.0.12 offers exciting new enhancements such as Jakarta EE 9.1 support, and configurable response headers, which offer more granular control over response headers. This release also provides several significant bug fixes.
```

Comment:
Jakarta EE 9.1 support is now available as part of Open Liberty 21.0.0.12, alongside configurable response headers, which offer more granular control over response headers!  Several significant bug fixes are also part of this release.